mk/config.mk

```diff
@@ -199,3 +199,24 @@ CFG_BOOT_SECONDARY_REQUEST ?= n
 
 # Default heap size for Core, 64 kB
 CFG_CORE_HEAP_SIZE ?= 65536
+
+# TA profiling.
+# When this option is enabled, OP-TEE can execute Trusted Applications
+# instrumented with GCC's -pg flag and will output profiling information
+# in gmon.out format to /tmp/gmon-<ta_uuid>.out (path is defined in
+# tee-supplicant)
+CFG_TA_GPROF_SUPPORT ?= n
+
+ifeq ($(CFG_TA_GPROF),y)
+# Build various user-mode libraries with profiling enabled (-pg)
+CFG_LIBUTEE_GPROF ?= y
```

Comment:
This seems complex, why not build all user TA libs with profiling support if `CFG_LIBUTEE_GPROF` is `y`?
If we discover that we need more flexibility, let's deal with that then.